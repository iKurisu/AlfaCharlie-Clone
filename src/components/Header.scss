@import 'sass/_variables.scss';

.header {
  position: absolute;
}

.header h1 {
  font-size: 10px;
  font-weight: 400;
  letter-spacing: 6px;
  color: $grey;
}

@media (max-width: $l) {
  .header {
    display: none;
  }
}

@media (min-width: $l) {
  .header {
    top: 43px;
    left: 46.75%;
  }
}

@media (min-width: $xxl) {
  .header {
    top: 63px;
  }

  .header h1 {
    font-size: 11px;
    letter-spacing: 6.65px;
  }
}
