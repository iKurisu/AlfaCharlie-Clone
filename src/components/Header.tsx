import React from "react";
import "./Header.scss";

const Header = (): JSX.Element => (
  <div className="header">
    <h1>SAN DIEGO CREATIVE AGENCY</h1>
  </div>
);

export default Header;
