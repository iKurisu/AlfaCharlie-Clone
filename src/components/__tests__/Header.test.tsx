import React from "react";
import renderer from "react-test-renderer";
import Header from "../Header";

it("renders correctly", (): void => {
  const header = renderer.create(<Header />);

  expect(header).toMatchSnapshot();
});
