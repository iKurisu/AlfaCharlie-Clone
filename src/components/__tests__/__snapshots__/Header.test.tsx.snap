// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="header"
>
  <h1>
    SAN DIEGO CREATIVE AGENCY
  </h1>
</div>
`;
